src/scripts/views/table.js

```diff
@@ -34,11 +28,12 @@ module.exports = Card.extend({
       this.$el.DataTable().clear().rows.add(this.collection.toJSON()).draw()
     // Otherwise, initialize the table
     } else {
-      // Map the array of columns to the expected format
-      var columns
+      var columnsPromise
 
+      // If columns specified in the config, use those
       if (this.config.columns) {
```

Comment:
I'm not even sure we should support this config option. If we do, it should probably work more like a whitelist (as @marks' `columnsToHide` option works like a blacklist) than an alternative to a fields collection. I left it in because that's probably out of scope of this PR.